stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1017,6 +1017,58 @@ LogicalResult ConvolutionOp::verify() {
       getResult().getType());
 }
 
+mlir::Speculation::Speculatability ConvolutionOp::getSpeculatability() {
+  // input_feature_dimension and kernel_input_feature_dimension must be static
```

Comment:
Could you split these comments preferably above each corresponding check?

Reply:
Good idea, done.